docs/install.md

```diff
@@ -12,9 +12,10 @@
 
 To be able to clone the repo and run the application you need:
 
--	[Node.js](http://nodejs.org/) and [NPM](https://www.npmjs.com/) installed. Here's a [handy installer](https://nodejs.org/dist/latest/) for Windows, Mac, and Linux. On Mac OSX using [brew]() is the easiest way to install `node` and `npm`.
+- Mac OS X or Linux. At this time Calypso does not run on Windows in development mode (but the [WordPress.com desktop app](https://desktop.wordpress.com/) supports Windows).
+-	[Node.js](http://nodejs.org/) and [NPM](https://www.npmjs.com/) installed. On Mac OS X using [brew](http://brew.sh/) is the easiest way to install `node` and `npm`.
 -	[Git](http://git-scm.com/). Try the `git` command from your terminal, if it's not found then use this [installer](http://git-scm.com/download/).
--	The repository also uses `make` to orchestrate compiling the JavaScript, running the server, and several other tasks. On Mac OSX, the easiest way to install `make` is through Apple's [Command Line Tools for Xcode](https://developer.apple.com/downloads/) (requires free registration).
+-	The repository also uses `make` to orchestrate compiling the JavaScript, running the server, and several other tasks. On Mac OS X, the easiest way to install `make` is through Apple's [Command Line Tools for Xcode](https://developer.apple.com/downloads/) (requires free registration).
```

Comment:
Nice fix - I just double-checked with [Apple](https://support.apple.com/kb/TA22541?locale=en_US) to make sure that this is the appropriate way to write Mac OS X and it looks like it's perfect :white_check_mark: